src/index.ts

```diff
@@ -16,11 +16,13 @@
 // ** https://github.com/googleapis/gapic-generator-typescript **
 // ** All changes to this file may be overwritten. **
 
+import * as v1 from './v1';
 import * as v1beta1 from './v1beta1';
+const BigQueryReadClient = v1.BigQueryReadClient;
 const BigQueryStorageClient = v1beta1.BigQueryStorageClient;
-export {v1beta1, BigQueryStorageClient};
+export {v1, BigQueryReadClient, v1beta1, BigQueryStorageClient};
 // For compatibility with JavaScript libraries we need to provide this default export:
 // tslint:disable-next-line no-default-export
-export default {v1beta1, BigQueryStorageClient};
+export default {v1, BigQueryReadClient, BigQueryStorageClient};
```

Comment:
@bcoe is that line right? I understand `export default` to be something that helps you prevent requiring destructuring assignment when doing a require.  If you export default with an object like this, wouldn't it just write over the properties with the same values we're exporting in the line above?

Possible my brain isn't working correctly

Reply:
Yeah, I think this is wrong. I had to manually update this and I shouldn't have added BigQueryReadClient here

Reply:
I think the original line was correct - just because this is what we had for JavaScript clients.

Wait wait wait wait!  I'm thinking... maybe drop it at all for the semver major?  (asked in the chat)

Reply:
decided to leave it here, but I'm still unsure about the right syntax about what to include

Reply:
Okay, I'll leave it and just export the v1 BigQueryReadClient